packages/mobile-top-nav/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@internetarchive/mobile-top-nav",
+  "version": "0.0.0",
+  "description": "Top Nav for Internet Archive that is optimized for the mobile experience",
+  "license": "AGPL-3.0-only",
+  "main": "index.js",
+  "module": "index.js",
+  "scripts": {
+    "start": "owc-dev-server --open ./index.html",
+    "lint:eslint": "eslint --ext .js,.html . --ignore-path .gitignore",
+    "format:eslint": "eslint --ext .js,.html . --fix --ignore-path .gitignore",
+    "lint:prettier": "prettier \"**/*.js\" --check --ignore-path .gitignore",
+    "format:prettier": "prettier \"**/*.js\" --write --ignore-path .gitignore",
+    "lint": "npm run lint:eslint && npm run lint:prettier",
+    "format": "npm run format:eslint && npm run format:prettier",
+    "test": "karma start --coverage",
+    "test:watch": "karma start --auto-watch=true --single-run=false",
+    "test:update-snapshots": "karma start --update-snapshots",
+    "test:prune-snapshots": "karma start --prune-snapshots",
+    "test:compatibility": "karma start --compatibility all --coverage",
+    "test:compatibility:watch": "karma start --compatibility all --auto-watch=true --single-run=false",
+    "test:bs": "karma start karma.bs.config.js --compatibility all --coverage",
+    "site:build": "npm run storybook:build",
+    "storybook": "start-storybook -p 9001",
+    "storybook:build": "build-storybook -o _site -s storybook-static"
+  },
+  "dependencies": {
+    "lit-element": "^2.2.1",
+    "lit-html": "^1.0.0"
+  },
+  "devDependencies": {
+    "@open-wc/demoing-storybook": "^0.3.0",
+    "@open-wc/eslint-config": "^1.0.0",
+    "@open-wc/polyfills-loader": "^0.2.1",
+    "@open-wc/prettier-config": "^0.1.10",
+    "@open-wc/testing": "^2.0.0",
+    "@open-wc/testing-karma": "^3.0.0",
+    "@open-wc/testing-karma-bs": "^1.0.0",
+    "@storybook/react": "^5.1.11",
+    "concurrently": "^4.1.2",
+    "es-dev-server": "^1.5.0",
+    "eslint": "^6.1.0",
+    "eslint-plugin-lit": "^0.5.0",
+    "gulp": "^4.0.2",
+    "lint-staged": "^8.0.0",
+    "owc-dev-server": "^0.1.2",
+    "polymer-cli": "^1.9.11",
+    "sinon": "^9.0.0",
+    "webpack-merge": "^4.1.5"
```

Comment:
I was getting complaints about things I hadn't addressed yet in my commits that prevented me from committing and pushing my work out. I like to at least make a WIP commit and push it up at the end of the day just in case, and Husky's commit hooks wouldn't allow that.

Reply:
Added back in.